eq-author-api/schema/typeDefs.js

```diff
@@ -285,6 +292,18 @@ type LatestDateValidationRule implements ValidationRule {
   availableMetadata: [Metadata!]!
 }
 
+type MinDurationValidationRule implements ValidationRule {
+  id: ID!
+  enabled: Boolean!
+  offset: Duration!
```

Comment:
I dislike using the an offset here. It isn't an offset. It is more like min or max value.

Reply:
It's definitely a `Duration` which we already described in our schema as:

```
type Duration {
  value: Int
  unit: DurationUnit!
}

enum DurationUnit {
  Days
  Months
  Years
}
```
What if we renamed `offset` to `duration`?

```
type MinDurationValidationRule implements ValidationRule {
  id: ID!
  enabled: Boolean!
  duration: Duration!
```

Reply:
:+1: Yes, that looks much better